deploy/group_vars/all/main.yml

```diff
@@ -185,6 +185,10 @@ restricted_group_name: admin
 
 # Research Environment
 install_odbc: True
+math_lm_server_name: research_gauss
+math_lm_math_id: math_id
+math_lm_activation_key: key
+math_lm_password: password
```

Comment:
@stirlingv These should go in `roles/math_lm/defaults/main.yml`, no?  Generally we only want to put stuff in "all" if it applies to multiple roles or servers.